gatling-http/src/main/scala/io/gatling/http/check/HttpCheck.scala

```diff
@@ -29,7 +30,7 @@ import io.gatling.http.response.{ Response, ResponseBodyUsageStrategy }
  * @param scope the part of the response this check targets
  * @param responseBodyUsageStrategy how this check uses the response body
  */
-case class HttpCheck(wrapped: Check[Response], scope: HttpCheckScope, responseBodyUsageStrategy: Option[ResponseBodyUsageStrategy])
+case class HttpCheck(wrapped: Check[Response], scope: Set[HttpCheckScope], responseBodyUsageStrategy: Set[ResponseBodyUsageStrategy])
```

Comment:
Sorry. I actually realize this is wrong.

For example, we only automatically add a status check when none is defined.
But consider the case where a checkIf sets a body check in one branch and a status one in the other one.
We would end up failing to set the default status check for the first branch!